### ISSUE TITLE
GetMany() doesn't support FetchStrategy?

### ISSUE THREAD

**Issue by GiganteX** [SharpRepository/SharpRepository#203]
Hi!

Tried to do this:

`List<Document> docs = repo.GetMany(id,fetch_strategy).ToList();
`
But it doesn't seem to be supported? Any reason or just not done yet? Will you implement it?

Best regards,
Gigante

**Comment by omarpiani** [comment]
This is the actual situation
```
IEnumerable<T> GetMany(params TKey[] keys);
IEnumerable<T> GetMany(IEnumerable<TKy> keys);
```

I can change in this:
```
IEnumerable<T> GetMany(params TKey[] keys);
IEnumerable<T> GetMany(IFetchStrategy<T> strategy, params TKey[] keys); // is usefull?
IEnumerable<T> GetMany(IEnumerable<TKy> keys, IFetchStrategy<T> strategy = null);
```

**Comment by omarpiani** [comment]
in next days :)

**Comment by omarpiani** [comment]
Pushed SharpRepository.Repository.2.0.4-beta1, take a look.
I hope that works without updating all the implementations

**Comment by omarpiani** [comment]
Released in 2.0.4.2. A feedback is wellcome.
